submissions/YuliiaDyka/frogger-game/app.js

```diff
@@ -0,0 +1,133 @@
+// Player constants
+const defaultPlayer = {
+    x: 204,
+    y: 409,
+};
+const stepSize = {
+    x: 102,
+    y: 84,
+}
+
+// Enemy constans
+const maxX = 500,
+      enemyStartX = -150,
+      affectedArea = 60,
+      numberofEnemies = 4;
+const enemyRows = [230, 146, 63, 313];
```

Comment:
I am tasked to add more rows where enemies may travel. I'd like to operate by categories of rows rather than pixels. Got no idea how to calculate the value for, say, row 4 or 5.
Please optimize this part.